fitness/models.py

```diff
@@ -123,6 +142,9 @@ def time_seconds(self):
             return time_delta.seconds
         return 0
 
+    def __str__(self):
+        return '{} in {} seconds'.format(self.time_seconds, self.distance) # maybe needs to change
```

Comment:
`return '{} in {} seconds'.format(self.distance, self.time_seconds) # maybe needs to change`